autonormalize/examples/example_data_gen.py

```diff
@@ -1,5 +1,5 @@
-import csv
-import random
+# import csv
```

Comment:
Do we need this file and the other files in the `examples` directory? Can we delete the directory or do we need to hang on to it for something?

Reply:
example3 is used in the tests. I deleted the others and just kept the code that generates example3. I could also git rid of the csv and make it a pytest fixture